README.md

```diff
@@ -93,6 +94,44 @@ and build the library with:
 npm run build-lib
 ```
 
+### Scripts For Development
+
+The following scripts can be used for development:
+
+#### Testing
+- `npm test`: runs the unit tests of the demo application and the lib's unit tests.
+- `npm test-lib`: runs the lib's unit tests in headless mode (no browser UI).
+- `npm test-lib-local`: runs the lib's unit test with the browser UI (for local development of the lib). 
+- `npm run webdriver-update`: installs Chrome webdriver in the required version for the E2E tests.
+- `npm run e2e`: runs the E2E tests from the demo application, see section [E2E Tests](README.md#Unit and E2E Tests).
+
+#### Building
+- `npm run build`: builds the lib in productive mode, see section [Building the Library](README.md#Building the Library).
+- `npm run build-dev`: builds the lib in development mode, see section [Building the Library](README.md#Building the Library).
+- `npm build-app`: builds the demo application in productive mode, see also section [Run the application in productive mode](README.md#Run the application in productive mode)
+- `npm run yalc-publish-lib`: builds the lib and publishes it locally using `yalc`.
+
+
+#### Unit and E2E Tests
+
+There are unit tests for the lib (`projects/dsp-ui/src`)
+and some basic unit tests for the demo application playground components (`src/app`). 
```

Comment:
As discussed in meeting, It would be good to explain further what the library (lib) means and what is the playground application?

Reply:
I added a section in https://github.com/dasch-swiss/dsp-ui-lib/pull/241/commits/3309d6b1f311fd4c20f67fd97b9cde3ba47d0914. Is that helpful?

The necessary build step is mentioned in section "First steps". Maybe this needs some more details.